spec/index.bs

```diff
@@ -2692,13 +2694,13 @@ has a default layout created and used instead.
 
             1. If |groupDescs|[|group|] has an entry |previousEntry| with {{GPUBindGroupLayoutEntry/binding}} equal to |binding|:
 
-                1. If |previousEntry| is equal to |entry| up to {{GPUBindGroupLayoutEntry/visibility}}:
+                1. If a member other than {{GPUBindGroupLayoutEntry/visibility}} or {{GPUBindGroupLayoutEntry/minBufferBindingSize}} is unequal between |previousEntry| and |entry|:
```

Comment:
I don't think we should mix these up. Let's have a separate list entry for the `minBufferBindingSize`

Reply:
Would something like this work? (closer to the original way it was stated)

- If visibility isn't equal
   -update visibility
- if min buffer binding size isn't equal
   -update min buffer binding size
- if any other property isn't equal
   -fail

Reply:
yes, but slightly different:
> if min buffer binding size isn't equal

Should be:

> if min buffer binding size is greater